rundeckapp/src/main/groovy/com/dtolabs/rundeck/app/api/tokens/Token.groovy

```diff
@@ -23,63 +23,69 @@ import com.dtolabs.rundeck.app.api.marshall.CollectionElement
 import com.dtolabs.rundeck.app.api.marshall.Ignore
 import com.dtolabs.rundeck.app.api.marshall.XmlAttribute
 import com.dtolabs.rundeck.core.authentication.tokens.AuthTokenMode
+import io.swagger.v3.oas.annotations.media.Schema
 import rundeck.AuthToken
 
 /**
  * @author greg
  * @since 3/23/17
  */
 @ApiResource
+@Schema
 class Token {
 
     @ApiVersion(37)
     @Ignore(onlyIfNull = true)
     @XmlAttribute
+    @Schema(description = "since: v37")
```

Comment:
I am not asking to remove it. I felt the `description` field seems redundant to `@ApiVersion(37)`.  Ideally, the meta description `since: v37` can be auto-generated with an annotation processor.

Reply:
Got it. It's not an issue. 

Actually, it's just my personal preference to avoid maintaining two copies of the same information so I don't have to check if they are in sync when editing it. 
There was a thought that it may be possible to create a custom annotation processor like Lombok to add this description at build time.